src/utils/isPlainObject.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {any} obj The object to inspect.
+ * @returns {boolean} True if the argument appears to be a plain object.
+ */
+export function isPlainObject(obj: any) {
```

Comment:
It's a shame that `immer` doesn't export their `isPlainObject` method, but maybe we could copy that over? It's a lot shorter.

https://github.com/immerjs/immer/blob/9007be0a68e8c521d23657729036d2eee9f5df83/src/utils/common.ts#L39-L44

Reply:
Use the one we have in RTK / React-Redux

Reply:
Thanks @markerikson ! I've updated to this, and opened a PR in RTK to export this so we can just use that. It's also debatable if we even need to do anything other than `typeof body === 'object'`, but I'd rather have more guardrails if the cost isn't really measurable.